### PR TITLE
Detailed TimeoutException message.

### PR DESCRIPTION
**Pull request by johnou**
In some cases we may want to know what actor calls are timing out.

### CODE REVIEW

File: actors/stage/src/main/java/com/ea/orbit/actors/runtime/MessageDefinitions.java [hunk 1]
```diff
@@ -40,4 +40,5 @@
     public static final Byte INTERFACE_ID = 0;
     public static final Byte METHOD_ID = 1;
     public static final Byte OBJECT_ID = 2;
+    public static final Byte METHOD_NAME_ID = 4;
```
Comment (johnou) [inline actors/stage/src/main/java/com/ea/orbit/actors/runtime/MessageDefinitions.java]: not sure why these are not primitives?

**Comment by DanielSperry** [comment]
Those are going away in the next version...
They are going out of the headers.

**Comment by JoeHegarty** [comment]
Further to Daniel's comments, we're going to reject this as it won't be neccesary in 0.6.x.
Please let us know if this continues to be a problem beyond then.

File: actors/stage/src/main/java/com/ea/orbit/actors/runtime/Execution.java [hunk 2]
```diff
@@ -1275,6 +1275,7 @@ private Throwable toSerializationSafeException(final Throwable notSerializable,
                 .withHeader(MessageDefinitions.INTERFACE_ID, actorReference._interfaceId())
                 .withHeader(MessageDefinitions.METHOD_ID, methodId)
                 .withHeader(MessageDefinitions.OBJECT_ID, ActorReference.getId(actorReference))
+                .withHeader(MessageDefinitions.METHOD_NAME_ID, m.getDeclaringClass().getName() + '.' + m.getName())
```
Comment (DanielSperry) [inline actors/stage/src/main/java/com/ea/orbit/actors/runtime/Execution.java]: We will try to avoid doing this.

**Pull request closed without merging**